<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>100</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Файл</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QSplitter" name="splitter_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="openButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Открыть...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Сохранить...</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Масштаб</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Коэфф.</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="scaleKoeffSpin">
         <property name="minimum">
          <double>-100000000000000000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>99999999999999991611392.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.200000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="scaleButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Масштаб.</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Поворот</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
          <property name="minimum">
           <double>-100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
          <property name="minimum">
           <double>-100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.990000000005239</double>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>-100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="setRotateButton">
         <property name="text">
          <string>Установить</string>
         </property>
        </widget>
        <widget class="QPushButton" name="rotateButton">
         <property name="text">
          <string>Повернуть</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Сдвиг</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSplitter" name="splitter_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>dX</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>-100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>dY</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>-100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>dZ</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>-100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="shiftButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Сдвинуть</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>391</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
